packages/pancake-uikit/src/components/Text/Text.tsx

```diff
@@ -23,6 +23,12 @@ const Text = styled.div<TextProps>`
   ${({ textTransform }) => textTransform && `text-transform: ${textTransform};`}
   ${space}
   ${typography}
+  ${({ theme, hasTooltip }) =>
+    hasTooltip &&
+    `
+    text-decoration: underline dotted ${theme.colors.textSubtle};
+    text-underline-offset: 0.1em;
+  `}
```

Comment:
This is coupling the Text component with Tooltip. I think it is better to create a component in the tooltip that uses `<Text />` or do it at the project level.

Reply:
For example - export a `TooltipText` component

Reply:
@hachiojidev hm.... how about just renaming the prop to something more generic like `underlined` or `dotted-underlined`? Seems a bit excessive to create a separate component for the same component with such little difference. 🤔

Reply:
I don't think it is excessive. This is a very specific style.
```
text-decoration: underline dotted ${theme.colors.textSubtle};
text-underline-offset: 0.1em;
    ```